virttest/migration.py

```diff
@@ -408,16 +408,19 @@ def _run_complex_func(vm, one_func, virsh_event_session=None):
             after_event = one_func.get('after_event')
             before_event = one_func.get('before_event')
             func = one_func.get('func')
+            wait_for_event_timeout = one_func.get('wait_for_event_timeout')
             if after_event and not virsh_event_session:
                 raise exceptions.TestError("virsh session for collecting domain "
                                            "events is not provided")
 
+            if not wait_for_event_timeout:
+                wait_for_event_timeout = '30'
```

Comment:
Can you plz explain these 2 lines?

Reply:
Seems equal with `wait_for_event_timeout = one_func.get('wait_for_event_timeout', '30')`

Reply:
We can use the get() method of a dict object and return a default value if a specific key doesn't exist. If the key exists and it's None, it'll be returned.

For example,
>>> one_func = {'before_pause': 'yes', 'need_sleep_time': '5', 'wait_for_event_timeout': None}
>>> one_func.get('wait_for_event_timeout', '30')
>>> one_func.get('non_exist', '30')
'30'

Reply:
Ok, so will you merge these 2 lines to 'wait_for_event_timeout = one_func.get('wait_for_event_timeout')' to make it simple?
BTW， is this 'timeout' value paired with 'after_event'? If yes, can we highlight 'after_event' in it's name?

Reply:
Updated, thanks.